dev/examples-to-tests.el

```diff
@@ -3,9 +3,12 @@
 (require 'dash-functional)
 
 (defun example-to-should (example)
-  (let ((actual (car example))
-        (expected (nth 2 example)))
-    `(should (equal ,actual ,expected))))
+  (-let [(actual sym expected) example]
```

Comment:
@fbergroth Would you write this without `-let` so we can merge it? Better safe than sorry :)